azure-pipelines.yml

```diff
@@ -280,3 +280,31 @@ jobs:
       inputs:
         pathtoPublish: '$(Build.ArtifactStagingDirectory)/SymbolPackages'
         artifactName:  Microsoft.Spark.Binaries
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - job: PackageWorker
+    dependsOn:
+      - Package
+    displayName: Package Microsoft.Spark.Worker
+    pool:
+      vmImage: ubuntu-16.04
```

Comment:
You can change this to just be: `pool: Hosted Ubuntu 16.04`

Reply:
Removed job